javaagent/build.gradle.kts

```diff
@@ -1,7 +1,7 @@
 import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
 import com.github.jk1.license.filter.LicenseBundleNormalizer
 import com.github.jk1.license.render.InventoryMarkdownReportRenderer
-import java.util.*
+import java.util.UUID
```

Comment:
@laurit sorry about that.  Is there an IDE setting I need to update to avoid this?

Reply:
I think IntelliJ is supposed to honor .editorconfig:

https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/5b5a31a0a8f15fc0d426c7f66b55716d77d429b8/.editorconfig#L70

Reply:
IntelliJ bug? 🤷‍♂️